conbench/entities/hardware.py

```diff
@@ -188,9 +188,27 @@ class MachineCreate(marshmallow.Schema):
 
 
 class ClusterCreate(marshmallow.Schema):
-    name = marshmallow.fields.String(required=True)
-    info = marshmallow.fields.Dict(required=True)
-    optional_info = marshmallow.fields.Dict(required=True)
+    name = marshmallow.fields.String(
+        required=True,
+        metadata={
+            "description": "Information related to cluster (e.g. `hosts`, `nodes` or `number of workers`)"
+            "configured to run a set of benchmarks. Used to differentiate between similar"
+            "benchmark runs performed on different sets of hardware"
+        },
+    )
+    info = marshmallow.fields.Dict(
```

Comment:
Only if you suggest my comment above — alternatively you can swap the descriptions if you want to keep the order of info, name, optional_info the same

```suggestion
    name = marshmallow.fields.Dict(
```